packages/taxdat/R/setup_helpers.R

```diff
@@ -824,5 +826,42 @@ check_od_param_sd_sd_prior_pooling <- function(obs_model,
   return(h_sd_sd_inv_od_updated)
 }
 
+#' check_mu_alpha
+#'
+#' @param x 
+#' @param default_value 
+#'
+#' @return
+#' @export
+#'
+check_mu_alpha <- function( x, 
+                            default_value = 0) {
+  
+  if (is.null(x)) {
+    par <- default_value
+  } else {
+    par <- try_conv_numeric(x)
```

Comment:
Is the `try_conv_numeric` function in the base R?  I don't see it anywhere... If it is in a package, we need to specify it explicitly. If possible, we should avoid adding new packages as dependencies in taxdat since this will require us to update lots of things